src/_common/modal/modal.ts

```diff
@@ -40,6 +40,8 @@ export default class AppModal extends Vue {
 		scroller: AppScrollScrollerTS;
 	};
 
+	@Emit('click-away') emitClickAway() {}
```

Comment:
I think we want to instead call this event "close" and emit it whenever the modal closes:
 - esc button pressed
 - backdrop clicked

Reply:
We were already emitting under those circumstances, so I'll change the naming to reflect that.